### ISSUE TITLE
DST Staging Review: Remove passive voice in "Examples" section

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#5596]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** va-tag-status
**Staging Review ticket:** [Staging Review: va-tag-status component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/129077)

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that aligns with VA.gov voice and tone content principles](https://design.va.gov/about/experience-standards/#user-encounters-content-that-aligns-with-vagov-voice-and-tone-content-principles)
**VA.gov Experience Standard - category:** [Credibility](https://design.va.gov/about/experience-standards/#category-credibility)
**This is an issue with the:** guidance
**High-priority:** No
**Collab Cycle Reviewer:** @kkitagawa-bosch (content), @jasonday (Accessibility)

### Description

In the examples section, the explanation paragraphs currently use passive voice. E.g., "Used to provide helpful information or something that warrants a user’s attention. Not used for negative consequences." 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="649" height="55" alt="Image" src="https://github.com/user-attachments/assets/f3b3c4e9-0ecb-44a7-ac02-0e28d34e7867" />

</details>

## Recommended action
This pattern could be improved both in terms of active voice and accessibility by including the subject in each sentence so screen reader users have immediate context. 

E.g., "Use the information status to provide helpful information or something that warrants a user’s attention. Do not use for negative consequences.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Plain language - Use active voice](https://design.va.gov/content-style-guide/plain-language/use-active-voice)
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
